pilot/model/service.go

```diff
@@ -427,3 +427,13 @@ func ParseLabelsString(s string) Labels {
 	}
 	return tag
 }
+
+// ExtractNamespaceFromHostname returns namespace from the (expected) hostname
+// string.
+func ExtractNamespaceFromHostname(hostname string) string {
```

Comment:
isn't this dangerous? how do we know if a hostname has a namespace or not?

Reply:
Ack. Removed.